internal/cluster/validations/validations.go

```diff
@@ -43,6 +44,10 @@ const (
 	dockerHubLegacyAuth             = "https://index.docker.io/v1/"
 	stageRegistry                   = "registry.stage.redhat.io"
 	ignoreListSeparator             = ","
+
+	// Size of file used to embed an Ignition config within a CPIO archive: 256 KiB
```

Comment:
This isn't quite accurate. I think something like this would be better:

`Size of the file used to embed an ignition config archive within an RHCOS ISO: 256 KiB`

Reply:
Just copied the comment from [coreos-assembler](https://github.com/coreos/coreos-assembler/blob/d2c968a1f3c75713a4e1449e3da657c5d5a5d7e7/src/cmd-buildextend-live#L113) :-D
But sure, sounds better.